### PR TITLE
Gtk source

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @EPashkin

### CODE REVIEW

**Comment by EPashkin** [comment]
No sourceview case in `fn docs`.

**Comment by GuillaumeGomez** [comment]
In `fn docs`?

**Comment by EPashkin** [comment]
Also in sourceview repo not last `gir-files` position

**Comment by EPashkin** [comment]
https://github.com/gtk-rs/lgpl-docs/blob/master/src/lib.rs#L24
Currently `cargo test --features embed-lgpl-docs` does nothing.

File: gtksource/docs.md [hunk 1]
```diff
@@ -0,0 +1,4039 @@
...
+
+## `line`
+a line number.
+## `category`
+category to search for, or `None`
+
+# Returns
+
+
+a newly allocated `glib::SList`.
```
Comment (EPashkin) [inline gtksource/docs.md:257]: Too many empty lines before?

File: gtksource/docs.md [hunk 2]
```diff
@@ -0,0 +1,4039 @@
...
+
+It can be convenient for showing a completion on-the-fly, without the need to
+add or remove providers to the `Completion`.
+
+Another solution is to add providers with
+`CompletionExt::add_provider`, and implement
+`CompletionProvider::match` for each provider.
+## `providers`
+
+a list of `CompletionProvider`, or `None`.
```
Comment (EPashkin) [inline gtksource/docs.md:553]: Not need line before? Or docs ignore it?

Comment (GuillaumeGomez) [reply gtksource/docs.md:553]: I don't think it's an issue.

**Comment by GuillaumeGomez** [comment]
Updated.

**Comment by EPashkin** [comment]
👍 Thanks, @GuillaumeGomez

**Pull request merged**